src/docs/quick-start/create-js-app/get-wrapper-metadata.md

```diff
@@ -36,7 +36,7 @@ const fetchHandler = async (event: FormEvent<HTMLFormElement>): Promise<void> =>
   }
   if (!Uri.isValidUri(uri)) {
     setManifest({
-      format: "0.0.1-prealpha.3",
```

Comment:
I don't think it's a good practice to have all of this inline code. I think it isn't very future-proof, and lacks context. Instead I would suggest:
- linking to the demo app
- linking to the codebase where this logic is happening

Reply:
In general I think we need to keep our docs brief and simple, and link to other sources if viewers want to dive deeper. Less = more

Reply:
I removed it. Thanks!